tests/test_models.py

```diff
@@ -30,12 +30,15 @@ def test_project_score_total_method(project, score_weights):
 
     # calculate the score "by hand" (within this test)
     total_score = 0
+    weights_sum = 0
     for index, score_field_name in enumerate(updated_scores):
         weight_field_value = getattr(score_weights, score_field_name)
+        weights_sum += weight_field_value
+        
         score_field_value = getattr(project_score_instance, score_field_name)
         total_score += score_field_value * weight_field_value
 
-    assert project_score_instance.total_score == total_score
+    assert int(project_score_instance.total_score) == int(total_score / weights_sum)
```

Comment:
this failed when trying to assert 3.8437500000000004 == 3.84375, so i cast to an int

in hindsight, this test seems kind of redundant and pointless since it's replicating the code that it's testing. let me know if you can think of a better way to test this?

Reply:
it's a fine type of test since the implementation might change.

checkout out this pytest method for dealing with just such issues: https://docs.pytest.org/en/latest/reference/reference.html?highlight=approx#pytest.approx